scss/boost_union/post.scss

```diff
@@ -154,14 +179,26 @@
     #searchform-navbar {
         background-color: $primary;
     }
+
+    /* Change the color of the active nav items's bottom border to white as this would be presented in the
+   primary color otherwise. */
+    .nav-link.active {
+        border-bottom-color: $white;
+    }
+
+    /* Change the color of the language selector icon to a light grey as well.
```

Comment:
Try to prepend this by

`/*csslint important:false*/`

I think you give the rationale to use this in the comment. The above line would then shut csslint quiet.
Here's a usage example: https://github.com/moodle/moodle/blob/MOODLE_402_STABLE/lib/yuilib/3.18.1/cssnormalize/cssnormalize.css#L565

Thanks for your contribution, @wa-bea !

Best,
Luca

Reply:
Please see my comment above :)